it/spring-boot-with-jib/src/main/java/io/fabric8/maven/sample/springboot/jib/HelloController.java

```diff
@@ -1,6 +1,6 @@
 package io.fabric8.maven.sample.springboot.jib;
 
-import org.springframework.web.bind.annotation.RequestMapping;
```

Comment:
You have removed the import but you haven't replaced the usage with `@GetMapping` (see line 7), that's why you're getting the error.

Reply:
@rohanKanojia ,Ok, I will fix this now.

Reply:
@rohanKanojia , On line 10 also, change to `RequestMapping` to `GetMapping` or not?